plugins/auth-backend-module-okta-provider/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@backstage/plugin-auth-backend-module-okta-provider",
+  "description": "The okta-provider backend module for the auth plugin.",
+  "version": "0.1.3",
```

Comment:
Quick question: I was expecting this version to be 0.0.0 since it is a new package, was this version generated automatically?